promscale/tobs/about.md

```diff
@@ -24,7 +24,6 @@ Currently this stack includes:
     long-term and allow analysis with both PromQL and SQL
 *   [TimescaleDB][timescaledb] for long term storage of metrics and provides
     ability to query metrics data using SQL
-*   [Promlens][promlens] tool to build and analyse promql queries with ease
 *   [Opentelemetry-Operator][opentelemetry-operator] to manage the lifecycle of       
     OpenTelemetryCollector Custom Resource Definition (CRDs)
 *   [Jaeger Query][jaeger-query] to visualize the traces
```

Comment:
This is not part of the current PR but since tobs no longer ships Jaeger query, we need to remove this line as well.

Reply:
Yes, we should remove the Jaeger Query component mention.